src/test/java/net/sf/jabref/logic/formatter/FormatterTest.java

```diff
@@ -85,6 +85,8 @@ public FormatterTest(Formatter formatter) {
                 new Object[]{new SuperscriptFormatter()},
                 new Object[]{new UnitFormatter()},
                 new Object[]{new RemoveBracesFormatter()}
+                // The EraseFormatter is special since it always returns null to indicate that the field should be deleted.
+                // new Object[]{new EraseFormatter()}
```

Comment:
Hm, what happens when a field is empty? Is it removed? I could live with an EraseFormatter if it would always return `""`. Does this make sense?

I do not have all the classes and their reponsibilities in mind for both the formatter and the cleanup. If you need more feedback, you will have to wait as I have to look this stuff up.